test/ParseableInterface/stored-properties-client.swift

```diff
@@ -12,7 +12,7 @@ import StoredProperties
 /// layout and use the right access patterns in the presence of a
 /// .swiftinterface file, in both resilient and non-resilient cases.
 
-// COMMON: %[[BAGOFVARIABLES:T16StoredProperties14BagOfVariablesV]] = type <{ %TSi, %TSb, [7 x i8], %TSi }>
+// COMMON: %[[BAGOFVARIABLES:T16StoredProperties14BagOfVariablesV]] = type <{ %TSi, %TSb, [{{[0-9]}} x i8], %TSi }>
```

Comment:
This is the thing we're supposed to be checking, though…can you do it as {{7|9}} or something like that? Or use FileCheck's fancy new -D feature?

Reply:
I think we'd want it to be {{3|7}}, because it's going to pad to word boundaries after a Bool.

Reply:
I just made up numbers, go for it.